update_languages_ios.sh

```diff
@@ -12,6 +12,7 @@ fi
 source supported_languages_mobile.sh
 
 ./update-translations.sh
+cp probe-mobile/en//Localizable.strings ${PROJDIR}/ooniprobe/Base.lproj/
```

Comment:
There is an extra `/` here

Reply:
> I would suggest instead of duplicating the same command to append `en` to the SUPPORTED_LANGUAGES array using the bash syntax:
> 
> ```
> SUPPORTED_LANGUAGES+=(en)
> ```

It wont work as the template for android (in the foor look) is dst_path="${PROJDIR}/app/src/main/res/values-${lang}/"
and what we want is 
"${PROJDIR}/app/src/main/res/values/

Reply:
Anyway we are not appending en, we are not touching the `SUPPORTED_LANGUAGES` at all to be able to operate outside the for loop

Reply:
Ah I understand what you mean.

The `en` language is actually a special case that requires difference treatment, hence it makes sense to keep it external from the rest of the supported languages flow.